src/actinia_api/swagger2/actinia_core/apidocs/project_management.py

```diff
@@ -30,10 +30,10 @@
     ProcessingResponseModel,
     SimpleResponseModel,
 )
-from actinia_api.swagger2.actinia_core.schemas.location_management import (
-    LocationListResponseModel,
+from actinia_api.swagger2.actinia_core.schemas.project_management import (
```

Comment:
You could update the year within the header of this file as well

Reply:
Totally right! I also found more in the header and also updated them!